pkg/reconciler/v1alpha1/testing/functional.go

```diff
@@ -422,6 +428,12 @@ func WithRouteLabel(key, value string) RouteOption {
 // ConfigOption enables further configuration of a Configuration.
 type ConfigOption func(*v1alpha1.Configuration)
 
+// WithConfigDeletionTimestamp will set the DeletionTimestamp on the Config.
+func WithConfigDeletionTimestamp(r *v1alpha1.Configuration) {
+	t := metav1.NewTime(time.Unix(1e9, 0))
```

Comment:
why not just time.Now()?
Same in the others.

Reply:
We end up calling in in a couple contexts and `Now()` drifts even in rapid succession.